packages/components/pages/f-loyalty/src/components/Loyalty.vue

```diff
@@ -123,9 +123,7 @@ export default {
     async mounted () {
         await this.init({
             brazeApiKey: this.brazeApiKey,
-            authToken: this.authToken,
-            logger: this.$log,
-            errorHandler: this.handleErrors
```

Comment:
Can we remove `handleErrors()` as well if we're going to remove all reference to it? Or is it worth wrapping the `await this.init()` call in a `try { ... } catch (e) { this.handleErrors(e); throw e; }` to help identify any other errors (i.e. events surfaced by the vue dev tools, even if they don't have a handler attached).

Reply:
just removing for now